target_hdfs/utils/hdfs.py

```diff
@@ -35,22 +36,20 @@ def get_hdfs_client() -> pa.fs.HadoopFileSystem:
 def get_hdfs_block_size() -> int:
     """Run the HDFS getconf command to get HDFS blocksize."""
     cmd = ["hdfs", "getconf", "-confKey", "dfs.blocksize"]
-    result = run(cmd, capture_output=True, text=True, check=True)
+    result = run(cmd, stdout=PIPE, stderr=DEVNULL, text=True, check=True)
     hdfs_block_size = int(convert_size_to_bytes(result.stdout.strip()))
     logger.info(f"HDFS block size: {hdfs_block_size} bytes")
     return hdfs_block_size
 
 
 def download_from_hdfs(source_path_hdfs: str, local_path: str) -> None:
     """Download a file from HDFS."""
-    logger.debug(f"Download file from HDFS: {source_path_hdfs} ")
-    pa.fs.copy_files(
-        source_path_hdfs,
-        local_path,
-        source_filesystem=get_hdfs_client(),
-        destination_filesystem=pa.fs.LocalFileSystem(),
-    )
-    logger.debug(f"File {source_path_hdfs} downloaded from hdfs to {local_path} ")
+    # Removing local temp file as hdfs -get command does not overwrite it
+    Path(local_path).unlink(missing_ok=True)
+    logger.info(f"Download file from HDFS: {source_path_hdfs} ")
+    cmd = ["hdfs", "dfs", "-get", source_path_hdfs, local_path]
+    run(cmd, stdout=DEVNULL, stderr=DEVNULL, check=True)
```

Comment:
I assume `pa` is PyArrow or something? If so that already has an hdfs client built in, why shell out to an additional JVM client to copy files?